sdc-static/src/lib/contours.js

```diff
@@ -6,7 +6,7 @@ function addObjectLayer(map, id, url, color, metadata) {
   // todo, use metadata properly
 
   // parameters to ensure the model is georeferenced correctly on the map
-  var multiplyZ = 0.000005
+  var multiplyZ = 0.000001
```

Comment:
Is this the max multiplyer? Please put thta in the comment, so we know why this number

Reply:
added comments and proper calculation